core/src/test/java/io/confluent/rest/RateLimitNetworkTrafficListenerTest.java

```diff
@@ -94,37 +119,91 @@ public void tearDown() throws Exception {
   @Test
   @DisplayName("NetworkTrafficRateLimitDisabled")
   public void testNetworkTrafficRateLimitDisabled_unlimited() throws Exception {
```

Comment:
This looks good. Can you add a comment here?
// All tests below run with 2 rest applications "external" & "internal". This is intentional to check the behaviour
// both apps are independently rate-limited. In the setup below -
// 1. "external" app's network rate-limits are enabled, disabled, & changed. 
// 2. "internal" app's network rate-limit is disabled(i.e. disabled).

Reply:
Done 👍